packages/database-mongo-sessions/src/mongo-sessions.ts

```diff
@@ -61,6 +69,23 @@ export class MongoSessions implements DatabaseInterfaceSessions {
       timestamps: { ...defaultOptions.timestamps, ...options.timestamps },
     };
 
+    if (typeof this.options.idProvider === 'function') {
+      this.options.idSessionProvider = this.options.idProvider;
+      console.warn(
+        `Deprecation Warning: The option "options.idProvider" is deprecated. 
+        Setting it to "options.idSessionProvider," please change your configuration, as "options.idProvider" will be removed in a future release.`
+      );
+    }
+
+    if (
+      typeof this.options.idSessionProvider === 'function' &&
+      this.options.convertSessionIdToMongoObjectId
+    ) {
+      console.warn(`You have set both "options.idSessionProvider" and "options.convertSessionIdToMongoObjectId = false" which will cause your "options.idSessionProvider" to be ignored. 
```

Comment:
I am a bit confused I think, shouldn't it be "options.convertSessionIdToMongoObjectId = true"?

Reply:
Yes! Good catch. Typo. Addressed in b89b6cdf77c759f9dc2dc7de936becf805184259